WSL/about.md

```diff
@@ -13,19 +13,19 @@ ms.assetid: 3cefe0db-7616-4848-a2b6-9296746a178b
 
 # Windows Subsystem for Linux Documentation
 
-The Windows Subsystem for Linux lets developers run Linux environments -- including most command-line tools, utilities, and applications -- directly on Windows, unmodified, without the overhead of a virtual machine.  
+The Windows Subsystem for Linux lets developers run GNU environment -- including most command-line tools, utilities, and applications -- directly on Windows, unmodified, without the overhead of a virtual machine.  
 
 You can:
 
 1. Choose your favorite Linux distributions [from the Windows Store](https://aka.ms/wslstore).
-1. Run common command-line utilities such as `grep`, `sed`, `awk`, etc. 
-1. Run Bash shell scripts and Linux command-line applications including:  
+1. Run common command-line free software such as `grep`, `sed`, `awk`, etc. 
+1. Run Bash shell scripts and GNU\Linux command-line applications including:  
     * Tools: vim, emacs, tmux
     * Languages: Javascript/node.js, Ruby, Python, C/C++, C# & F#, Rust, Go, etc.
     * Services: sshd, MySQL, Apache, lighttpd
-1. Install additional Linux tools using the distribution's built in package manager (`apt-get`, for example).
-1. Invoke Windows applications from the Linux console.
-1. Invoke Linux applications on Windows.
+1. Install additional GNU\Linux free software using the distribution's built in package manager (`apt-get`, for example).
+1. Invoke Windows applications from the Bash console.
```

Comment:
Broader than just Bash. This should probably be "a Linux terminal"

Reply:
Note: Terminals are just the apps that accept input and display output. It's the shell that launches the app. Perhaps "Invoke Windows applications from the Linux shell/command-line"?

Reply:
Maybe "Invoke Windows applications from the GNU/Linux shell" ?